src/test/regress/expected/failure_1pc_copy_append.out

```diff
@@ -0,0 +1,265 @@
+--------------+---------+--------------+---------------+---------------+---------+------------+-------------+-----------+----------+-------------
+ copy_test    |  100400 | t            | 0             | 3             |  100400 |          1 |        8192 | localhost |    57637 |         100
+ copy_test    |  100400 | t            | 0             | 3             |  100400 |          1 |        8192 | localhost |    57640 |         101
+(2 rows)
+
+SELECT count(1) FROM copy_test;
+ count 
+-------
+     4
+(1 row)
+
+---- kill the connection when we try to get the size of the table ----
+SELECT citus.mitmproxy('conn.onQuery(query="pg_table_size").kill()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+COPY copy_test FROM PROGRAM 'echo 0, 0 && echo 1, 1 && echo 2, 4 && echo 3, 9' WITH CSV;
+WARNING:  server closed the connection unexpectedly
+	This probably means the server terminated abnormally
+	before or while processing the request.
+CONTEXT:  while executing command on localhost:57640
+WARNING:  connection not open
+CONTEXT:  while executing command on localhost:57640
+ERROR:  failure on connection marked as essential: localhost:57640
+SELECT * FROM pg_dist_shard s, pg_dist_shard_placement p
+  WHERE (s.shardid = p.shardid) AND s.logicalrelid = 'copy_test'::regclass
+  ORDER BY placementid;
+ logicalrelid | shardid | shardstorage | shardminvalue | shardmaxvalue | shardid | shardstate | shardlength | nodename  | nodeport | placementid 
+--------------+---------+--------------+---------------+---------------+---------+------------+-------------+-----------+----------+-------------
+ copy_test    |  100400 | t            | 0             | 3             |  100400 |          1 |        8192 | localhost |    57637 |         100
+ copy_test    |  100400 | t            | 0             | 3             |  100400 |          1 |        8192 | localhost |    57640 |         101
+(2 rows)
+
+SELECT count(1) FROM copy_test;
+ count 
+-------
+     4
+(1 row)
+
+-- we round-robin when picking which node to run pg_table_size on, this COPY runs it on
+-- the other node, so the next copy will try to run it on our node
+COPY copy_test FROM PROGRAM 'echo 0, 0 && echo 1, 1 && echo 2, 4 && echo 3, 9' WITH CSV;
+---- kill the connection when we try to get the min, max of the table ----
+SELECT citus.mitmproxy('conn.onQuery(query="SELECT min\(key\), max\(key\)").kill()');
+ mitmproxy 
+-----------
+ 
+(1 row)
+
+COPY copy_test FROM PROGRAM 'echo 0, 0 && echo 1, 1 && echo 2, 4 && echo 3, 9' WITH CSV;
+WARNING:  server closed the connection unexpectedly
+	This probably means the server terminated abnormally
+	before or while processing the request.
+CONTEXT:  while executing command on localhost:57640
+WARNING:  connection not open
+CONTEXT:  while executing command on localhost:57640
+ERROR:  failure on connection marked as essential: localhost:57640
```

Comment:
Pretty minor: this test took some time for me to digest. Since we've not printed placements just after the successful `COPY` operation 2 lines before, I thought `oh, we're erroring out but also creating the placements successfully, what's happening` 😄 

So, could be useful to print the placements just before starting the test as well.

Reply:
good point, added!